laterpay/__init__.py

```diff
@@ -472,3 +482,78 @@ def get_access(self, article_ids, product_key=None):
             raise Exception(data['status'])
 
         return data
+
+    def get_request_headers(self):
+        """
+        Return a ``dict`` of request headers to be sent to the API.
+        """
+        return {
+            'X-LP-APIVersion': 2,
+            # TODO: Automatic versioning.
+            'User-Agent': 'LaterPay Client - Python - v4.2.0'
+        }
+
+    def get_access_url(self):
+        """
+        Return the base url for /access endpoint.
+
+        Example: https://api.laterpay.net/access
+        """
+        return self._access_url
+
+    def get_access_params(self, article_ids, lptoken=None):
+        """
+        Return a params ``dict`` for /access call.
+
+        A correct signature is included in the dict as the "hmac" param.
+
+        :param article_ids: list of article ids or a single article id as a
+                            string
+        :param lptoken: optional lptoken as `str`
+        """
+        if not isinstance(article_ids, (list, tuple)):
+            article_ids = [article_ids]
+
+        lptoken = str(lptoken or self.lptoken)
+
+        params = {
+            'cp': self.cp_key,
+            'ts': str(int(time.time())),
+            'lptoken': lptoken or self.lptoken,
+            'article_id': article_ids,
+        }
+
+        params['hmac'] = signing.sign(
+            secret=self.shared_secret,
+            params=params.copy(),
+            url=self._access_url,
```

Comment:
Maybe we should use `.get_access_url()` here also? Or allow to pass that url as another argument in this method?